src/Symfony/Bridge/Twig/Extension/WorkflowExtension.php

```diff
@@ -64,6 +65,16 @@ public function getEnabledTransitions(object $subject, string $name = null): arr
         return $this->workflowRegistry->get($subject, $name)->getEnabledTransitions($subject);
     }
 
+    /**
+     * Returns a transition or null.
+     *
+     * @return Transition|null Enabled transition asked for
+     */
+    public function getEnabledTransition(object $subject, string $transition, string $name = null): ?Transition
+    {
+        return $this->workflowRegistry->get($subject, $name)->getEnabledTransition($subject, $transition);
```

Comment:
this means the bridge is now incompatible with workflow < 5.2, this should be reflected in the composer.json file, section "conflict" & "require-dev". That's the reason why deps=low tests fail.

Reply:
:pray:  Thank you.
I made the requested changes. Is it ok for you ?

I still have this error : ```1) Symfony\Component\Messenger\Bridge\Doctrine\Tests\Transport\DoctrineReceiverTest::testOccursRetryableExceptionFromConnection

Error: Class 'Doctrine\DBAL\Driver\PDOException' not found```

But I guess it has nothing to do with the Workflow component ^^